tests/Integration/Behaviour/Features/Scenario/Carrier/carrier_management.feature

```diff
@@ -175,26 +225,219 @@ Feature: Carrier management
       | active       | true                               |
       | delay[en-US] | Shipping delay new                 |
 
+Scenario: Partially editing carrier with handling shipping
+    When I create carrier "carrier1" with specified properties:
+      | name             | Carrier 1                          |
+      | grade            | 1                                  |
+      | trackingUrl      | http://example.com/track.php?num=@ |
+      | position         | 2                                  |
+      | active           | true                               |
+      | delay[en-US]     | Shipping delay                     |
+      | shippingHandling | false                              |
+      | isFree           | false                              |
+      | shippingMethod   | 1                                  |
```

Comment:
For a more human-friendly test (which is the purpose of gherkin), you should use some strings here that represent the values like in this case: `default` `weight` `price` `free`

It's the behat context step that then transforms the strings into the appropriate integer const values